deploy/charts/harvester/Chart.yaml

```diff
@@ -46,3 +46,7 @@ dependencies:
     version: 5.0.32
     repository: file://charts/minio
     condition: minio.enabled
+  - name: harvester-network-controller
+    version: 0.1.0
```

Comment:
This is inconsistent with the versions in `harvester-network-controller/Chart.yaml`